registry/flutter_gallery.test

```diff
@@ -0,0 +1,6 @@
+contact=raboughanem@google.com
+contact=perc@google.com
+fetch=git -c core.longPaths=true clone https://github.com/flutter/gallery.git tests
+fetch=git -c core.longPaths=true -C tests checkout 0423ac456c40a913a9750844e6ed9fa2c9a3a45e
```

Comment:
Would the commit hash need to be updated?

Reply:
Updated to https://github.com/flutter/gallery/commit/8542ca8beebc8e989adee160f7f1f0841a734db2